hydra_agent/partial_collection_crawler.py

```diff
@@ -0,0 +1,95 @@
+from requests import get
+
+
+class PartialCollectionCrawler:
+    """
+    Crawler for Crawling Collections. 
+    It can move forwards, backwards or can jump to specific page
+    """
+
+    def __init__(self, response):
+        self.response = response
+        self.base_url = 'http://localhost:8080'
```

Comment:
The base_url should not be hardcoded. You can pass one more argument in __init__ for base_url and set its default to localhost.